tests/simulation/physics/sources/test_temporal_profiles.py

```
@@ -0,0 +1,222 @@
+    arrays = _run(objects, constraints, config)
+
+    p = complex(arrays.detector_states["phasor"]["phasor"][0, 0].ravel()[0])
+    assert abs(p) > 1e-10, f"CW phasor amplitude should be nonzero, got |p|={abs(p):.2e}"
+
+
+def test_gaussian_pulse_broadband():
+    """GaussianPulseProfile spectral amplitude follows the Gaussian envelope.
+
+    GaussianPulseProfile produces the time-domain signal::
+
+        E(t) = exp(-((t - t0)² / (2σ_t²))) · cos(2π f_c t),   t0 = 6σ_t
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `σ` (GREEK SMALL LETTER SIGMA). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF002)